### PR TITLE
webui: minor visual changes in schedule maintenance window and its modal

### PR DESCRIPTION
**Pull request by aldemira**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by aldemira** [comment]
Here are the before-after screenshots of the changes:

screen shot 2017-07-03 at 13 04 06
screen shot 2017-07-03 at 13 04 31
screen shot 2017-07-03 at 13 04 40
screen shot 2017-07-03 at 13 04 49

**Comment by aldemira** [comment]
Mind you, top modal window screenshot is the new one.

**Comment by murrant** [comment]
@aldemira guessed that by the ridiculous no/yes layout 😊

File: html/pages/alert-schedule.inc.php [hunk 1]
```diff
@@ -55,11 +56,16 @@
...
+        },
+        "status": function(column, row)
+        {
             if (row.status == 1) {
-                response = response + '<button type="button" class="btn btn-xs btn-danger" disabled>Lapsed</button>';
+                //response = '<button type="button" class="btn btn-xs btn-danger" disabled>Lapsed</button>';
+                response = '<span class="label label-danger" disabled>Lapsed</span>';
             } else if (row.status == 2) {
-                response = response + '<button type="button" class="btn btn-xs btn-success" disabled>Current</button>';
+                //response = '<button type="button" class="btn btn-xs btn-success" disabled>Current</button>';
```
Comment (murrant) [inline html/pages/alert-schedule.inc.php]: Please remove the comments here.

**Comment by murrant** [comment]
Looks good, but while testing this, I noticed the alerts on this page never go away.

Create a window (omit mapped device) add it and save.  Now create a new window, alert is still in the modal.

Also, the "Scheduling maintenance ok" alert never goes away after adding a window.

**Comment by aldemira** [comment]
OK, I see what you mean. I'll try to fix that.

**Comment by aldemira** [comment]
For the future schedules (status == 0) there are no nice labels, so I've named it "Set" with an info-warning, but we can rename it as incoming or something else.

**Comment by laf** [comment]
Thanks as always @aldemira

**Pull request merged**